### DISCUSSION TITLE
How to install rtl8187 driver ?

### DISCUSSION THREAD

**Discussion by MikeWIseX0** [termux/termux-app#3463] [General]
Hello I want to install an rtl8187 driver on termux I asked chat gpt about it and he gave me the aircrack-ng/rtl8187 (git clone git@github.com:aircrack-ng/rtl8187.git
) repository it gave me an error after asking what's my username and password (which was given correctly) that it does not support this function anymore (the username and password one) so gpt gave me another idea to install openssh and configure an ssh key for github although being successful it still didn't work giving me an error that the repository doesn't exist here gpt actually gave up and started telling me to go see a community forum about it and here I am can anyone tell me how to install the driver if possible ? And if not why ? Thanks in advance

**Comment by sylirre** [comment]
You need to compile a custom kernel. Nethunter provides a hint for doing this: https://www.kali.org/docs/nethunter/nethunter-kernel-4-config-3/

Don't use ChatGPT if literally know nothing about the topic. It will confuse you even more. The ChatGPT can be an assistant, but not a replacement for a brain. GPT system is just a neural network based text generation algorithm. It can't think and verify whether the answer is correct.

And finally we do not help with setting up hacking software like `aircrack-ng`:

* https://wiki.termux.com/wiki/Hacking
* https://wiki.termux.com/wiki/Community_Rules
